egs/swbd/s5c/RESULTS

```diff
@@ -156,6 +156,14 @@ exit 0
 %WER 12.1 | 1831 21395 | 89.1 7.1 3.8 1.3 12.1 48.1 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_fsh_fg/score_12_0.0/eval2000_hires.ctm.swbd.filt.sys
 %WER 13.6 | 1831 21395 | 87.9 8.2 3.9 1.5 13.6 51.0 | exp/nnet3/tdnn_sp/decode_eval2000_hires_sw1_tg/score_11_0.0/eval2000_hires.ctm.swbd.filt.sys
 
+# results with nnet3 cnn+tdnn: local/nnet3/run_tdnn.sh (1.2.2016) (2 epoch training on speed-perturbed and volume perturbed data)
```

Comment:
Did you manage to run the TDNN experiment where you match the number of updatable layers  and parameters between CNN and TDNN models ? That is not necessary for this PR but good to know if you intend to report these results.